back/engines/commercial/public_api/spec/acceptance/v2/ideas_spec.rb

```diff
@@ -61,14 +74,22 @@
       in: :query
     )
 
+    parameter(
+      :type,
+      'Filter by type of idea - idea or survey - returns both by default',
+      required: false,
+      type: :string,
+      in: :query
+    )
```

Comment:
```suggestion
    parameter(
      :type,
      'Filter by type of idea – idea or survey – returns both by default',
      required: false,
      type: :string,
      enum: %w[idea survey]
      in: :query
    )
```

Ideally, we should get the list of possible values from a service or the controller.